datasets/restaurant.py

```diff
@@ -36,7 +36,7 @@ def restaurant_script_iter(self) -> Iterator[TestAction]:
             "When I talk to you as the waiter ('Waiter: what will it be sir?'), then you will reply as if you were the "
             "customer at a restaurant. Give straight answers to the questions and avoid going off script. Understood?"
         )
-        yield self.wait(percentage_finished=20)
+        yield self.wait(tokens=self.memory_span * .2)
```

Comment:
I like this alternative. It could be extracted as `self.wait_percentage(20)` for improved ease of use.

Reply:
This is a little dangerous, as its a wait for 20% of the memory span, not until 20% of the span has elapsed.

Reply:
Ah, you're right. Yeah, in that case it would be nice to have a shortcut for waiting until a percentage of the memory span has elapsed 👍🏾